apps/electron-app/src/main/browser/overlay-manager.ts

```
@@ -0,0 +1,1061 @@
+          break;
+      }
+    }
+  }
+
+  /**
+   * Merge multiple content commands into one
+   */
+  private mergeContentCommands(commands: OverlayCommand[]): OverlayContent {
+    return commands.reduce((merged, cmd) => {
+      if (cmd.data) {
+        return { ...merged, ...cmd.data };
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)